### ISSUE TITLE
[desktop] Support screenreaders

### ISSUE THREAD

**Issue by twiss** [codexa/firetext#213]
I don't know what existing editors do, but we should do that.

I've never used one, but I imagine it would be most important to read the names of the documents on the welcome screen, and the contents of the documents themselves.

Also, css supports highlighting what the screenreader is currently reading.

Spin-off from https://github.com/codexa/firetext/issues/206#issuecomment-62367711

**Comment by ferndot** [comment]
That is a good idea. Do you think that this would require any visual changes? Or, is speech accomplished through invisible CSS properties and HTML attributes?

**Comment by Ryuno-Ki** [comment]
It basically comes to down to add ARIA-tags where it is necessary.

My device does not have that version, but CodingFree reported about [how to activate the screenreader](http://www.codingfree.com/2014/11/how-to-use-screen-reader-in-firefox-os.html) a few weeks ago. Aside that, you can reach out to [Marco Zehe](http://www.marcozehe.de/) for help (he's a Mozillian, too).

I look after compiling a list of Go-To-documents for consulting.

**Comment by Ryuno-Ki** [comment]
Are you familiar with `accesskey`-attribute? I found [an abstract on WebAIM](http://webaim.org/techniques/keyboard/accesskey). I'd like to see a solution, which listens to `keydown` events (since old IE may not fire `keypressed`), but passes the event forward. It's so annoying on certain sites, that the app steals the focus, so I cannot use shortcuts like Ctrl-T to open a new tab.

Oh, and on MDN, there's a [checklist](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Mobile_accessibility_checklist) :)
